### PR TITLE
System PK index wasn't getting created in all circumstances

### PR DESCRIPTION
**Pull request by rjmac**
..leading to slow updates on large datasets

This isn't in `optimize` because that pretty much assumes it's creating
userspace indexes.

### CODE REVIEW

**Comment by rjmac** [comment]
retest this please

File: store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala [hunk 1]
```diff
@@ -551,6 +551,9 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
         loader.flush()
       }
     }
+    truthSchema.values.find(_.isSystemPrimaryKey).foreach { pkCol =>
+      sLoader.makeSystemPrimaryKey(pkCol)
+    }
```
Comment (chitang) [inline store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala]: Anything needs to be done if the dataset has custom user key?

Comment (rjmac) [inline store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala]: No, AFAICT the soql-pg-adaptor doesn't do anything special for user keys at all.

**Comment by chitang** [comment]
lgtm

**Pull request merged**